### PR TITLE
Added language documentation

### PR DESCRIPTION
**Pull request by leophys**
As previous pull request, with the suggested changes.

### CODE REVIEW

File: doc/source/languages.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+a 2 column layout. On the left you will find the english strings. You will
+be asked to fill the correspondig translation on the right.
+Once finished, you can save the file and either make a pull request, if you
+cloned our repository from github.com, or send us back the files.
+
+.. note::
+    If you cloned the repository from github.com, before making the pull
+    request, make sure to create a folder with path
+    ``webant/translations/<two letter code of the language>/LC_MESSAGES/``
+    and put the translation ``message.po`` file inside it.
```
Comment (boyska) [inline doc/source/languages.rst]: it is `messages.po` not `message`

File: doc/source/languages.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+See below for more details.
+
+If you are NOT confident with github.com
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Waiting for an easier solution, you can e-mail us at ``insomnialab@hacari.org``.
+We will send you a ``messages.po`` file that you will easily be able to modify
+with the following tool.
+
+What tools do you need
```
Comment (boyska) [inline doc/source/languages.rst]: the "tools" is actually just poedit? Isn't it simpler to just state it in a less generic way?

File: doc/source/languages.rst [hunk 3]
```diff
@@ -0,0 +1,67 @@
+Languages
+=========
+
+A program can be a very useful tool, but many times one is expected
+to give access to it to many people who are not native English 
+speakers.
+
+Libreant makes use of a specific module of Flask (Flask-Babel) which
```
Comment (boyska) [inline doc/source/languages.rst]: is this paragraph useful, for a non-developer?

File: doc/source/index.rst [hunk 4]
```diff
@@ -18,7 +18,7 @@ Contents:
    documentation
    developer
    api/modules
-
+   languages
```
Comment (boyska) [inline doc/source/index.rst]: the order is still wrong

File: doc/source/languages.rst (hunk 3 above)
Comment (leophys) [inline doc/source/languages.rst]: I made the correction you suggested.

On 27/02/2015 17:56, BoySka wrote:
>> @@ -0,0 +1,67 @@
>> +Languages
>> +=========
>> +
>> +A program can be a very useful tool, but many times one is expected
>> +to give access to it to many people who are not native English 
>> +speakers.
>> +
>> +Libreant makes use of a specific module of Flask (Flask-Babel) which
> is this paragraph useful, for a non-developer?
>
It is just an introduction. I think it is just the right place to
introduce the module we used.

Comment (boyska) [inline doc/source/languages.rst]: On Fri, Feb 27, 2015 at 09:06:43AM -0800, Blallo wrote:
>I made the correction you suggested.

I still see `message.po`. Apart from this, I'd merge.

>> is this paragraph useful, for a non-developer?
>>
>It is just an introduction. I think it is just the right place to
>introduce the module we used.

ok, no problem

**Comment by ael-code** [comment]
I think ```internalization``` suites better than ```languages``` as the file name

**Comment by boyska** [comment]
>I think ```internalization``` suites better than ```languages``` as the file name

you probably mean 'internationalization' :)

**Comment by leophys** [comment]
On 02/28/2015 01:22 PM, BoySka wrote:
> you probably mean 'internationalization' :)
> 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/insomnia-lab/libreant/pull/114#issuecomment-76523931
> 

Tried to answer to all your doubts. Hope it's enough for merge :)

**Comment by ael-code** [comment]
I'm not sure that the translation procedure you provided is the right one. We need to discuss.

**Comment by boyska** [comment]
I don't remember what's the status of this. @ael-code can you remember us the questions you raised?

**Comment by boyska** [comment]
@ael-code ping

**Comment by ael-code** [comment]
- One problem was about naming ```message.po``` catalog file of different languages will led to confusion. ( see the mail process )
- It is explained only the process to introduce new language but not the one to update an already existing language.
- I'm also concerned about initialize new language support manually without using ```pybabel init```. In this doc we suggest to create manually the new language catalog.

**Comment by ael-code** [comment]
@leophys if you aren't going to work on this, please close it.
Anyway I suggest you to keep the changes, they will be useful for future elaboration.

**Comment by leophys** [comment]
On 06/08/2015 01:34 PM, ael wrote:
> @leophys if you aren't going to work on this, please close it.
> Anyway I suggest you to keep the changes, they will be useful for future elaboration.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/insomnia-lab/libreant/pull/114#issuecomment-109958309
> 

@ael-code
I vaguely remember that the pullreq you are talking about was rejected
also because we wanted (and succeeded?) in changing the model of
localization, relying on some external service I don't remember.
Can you update me. I could eventually keep on working and submit an
acceptable pullreq.

**Pull request closed without merging**